lgsm/config-default/config-lgsm/rwserver/_default.cfg

```diff
@@ -154,7 +152,8 @@ glibc="null"
 ## Game Server Directories
 systemdir="${serverfiles}"
 executabledir="${serverfiles}"
-executable="java -Xmx${javaram}m -jar ${serverfiles}/server.jar"
+prexecutable="java -Xmx${javaram}M -jar"
```

Comment:
With the new variable being there the javaram variable will not be updated will be again a problem.
We need probably as well for this a eval.